models.go

```diff
@@ -68,10 +68,10 @@ type ChatCompletionRequest struct {
 	MaxTokens int `json:"max_tokens,omitempty"`
 
 	// (-2, 2) Penalize tokens that haven't appeared yet in the history.
-	PresencePenalty float32 `json:"presence_penalty,omitempty"`
+	PresencePenalty *float32 `json:"presence_penalty,omitempty"`
```

Comment:
it looks like `PresencePenalty` and `FrequencyPenalty` default to `0` in the [API](https://platform.openai.com/docs/api-reference/completions/create#completions/create-presence_penalty). so we could potentially not change these to avoid breaking existing code, although I do agree it would be more appropriate to not override any changing defaults later in the API itself.